lib/scraping/problem_checker.rb

```diff
@@ -0,0 +1,51 @@
+class ProblemChecker
+  @human_verified_time = 365
+  @outdated_time = 183
+
+  def data_check_routine
+    check_empty_fields(Person)
+    check_empty_fields(Room)
+    check_for_outdated(Room)
+  end
+
+  def check_empty_fields(model)
+    model.all.each do |entry|
+      entry.attributes.each do |name, value|
+        save_problem("missing", entry, name) unless value || name["human_verified"]
+      end
+    end
+  end
+
+  def check_for_outdated(model)
+    model.all.each do |entry|
+      save_problem("outdated", entry, "all") if entry.updated_at.days_since(@outdated_time).past?
+    end
+  end
+
+  def check_for_conflict(entry, new, field)
```

Comment:
I think this method can be simplified by using a guard clause. 
The only case it should return `false` is when there is no entry yet (`old.nil? == true`).
Otherwise `true` is returned which could be added at the end of the method. Thereby every other return statement could be removed.

Reply:
But what about the case we run the scraper and the value is different, human verified is not set and the last update is older than a day? Then the current old should be overwritten. Your suggestion does not reflect that.